mlflow/tensorflow/__init__.py

```diff
@@ -553,6 +699,32 @@ def predict(self, data):
         else:
             return pandas.DataFrame.from_dict(data=pred_dict)
 
+class _TF2ModuleWrapper:
+    def __init__(self, model):
+        self.model = model
+
+    def predict(self, data):
+        return self.model(data)
```

Comment:
Do we need to do any conversion of `data` from numpy / list, or can `self.model()` operate on these types directly?

Reply:
I don't think so.
The self.model() can support any type input, it depends on how user implement `model.__call__` method.